src/layouts/app/sidebar-mixin.ts

```diff
@@ -24,7 +24,7 @@ export default (superClass: Constructor<LitElement & HassBaseEl>) =>
       super.firstUpdated(changedProps);
       this.addEventListener("hass-dock-sidebar", (ev) => {
         this._updateHass({ dockedSidebar: ev.detail.dock });
-        storeState(this.hass);
+        storeState(this.hass!);
```

Comment:
Why?

Reply:
Because `this.hass` may be undefined, use `!` to avoid typescript complaint. It is not a concern since above `this._updateHass` will assign `this.hass` anyway